packages/api-providers/src/web3/vanchor-withdraw.ts

```diff
@@ -156,6 +158,7 @@ export class Web3VAnchorWithdraw extends VAnchorWithdraw<WebbWeb3Provider> {
         );
 
         console.log('root of recreated merkle tree', testMerkleTree.root().toHexString());
+        console.log('root of merkle tree on source configured on the dest chain: ', destHistorySourceRoot);
```

Comment:
Do we need these logs?

Reply:
Would suggest making it structured if we keep logs in the dApp also.

Something like `packageName::fileName: <log contents>`

Reply:
Since these console logs are useful for development, I would like to leave them in - rather than having to add/remove in every PR.

I think removing logs that appear repeatedly such as in component renders is important for not cluttering the console, but something on an api-provider action is worthwhile to leave in.

As for the log prefix, I don't think it's particularly useful. The console will output the filename, and devs can easily ctrl+F on the output to find exact lines if desired.